src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -1200,7 +1200,7 @@ static QString quotedList( const QVariantList &list )
     }
 
     QString inner = i->toString();
-    if ( inner.startsWith( '{' ) )
+    if ( inner.startsWith( '{' ) || i->type() == QVariant::Int || i->type() == QVariant::LongLong )
```

Comment:
A bit too late since master has been merged already, what about `UInt`?

Reply:
I think we never deal with a Uint QVariant list in postgres provider. It's not a possible [fieldtype](https://github.com/qgis/QGIS/blob/master/src/providers/postgres/qgspostgresprovider.cpp#L987), so I would say we are fine